kotlin/choose-camera-controller/src/main/java/com/esri/arcgisruntime/sample/choosecameracontroller/MainActivity.kt

```diff
@@ -49,6 +49,8 @@ class MainActivity : AppCompatActivity() {
   private lateinit var plane3d: Graphic
   private lateinit var orbitLocationCameraController: OrbitLocationCameraController
   private lateinit var orbitPlaneCameraController: OrbitGeoElementCameraController
+  // objects that implement Loadable must be class fields to prevent being garbage collected before loading
+  private val modelSceneSymbol: ModelSceneSymbol by lazy { loadModel() }
```

Comment:
not required since symbol is put on graphic and graphic onto graphics overlay outside of a done loading listener